opsdroid/helper.py

```diff
@@ -10,7 +10,11 @@
 
 
 def get_opsdroid():
-    """Return the running opsdroid instance."""
+    """This method return an opsdroid instance
```

Comment:
Keep the original docstring text "Return the running opsdroid instance." then add a blank line and include your Returns: object...